lib/gps_sim/Kconfig

```diff
@@ -67,9 +71,16 @@ config GPS_SIM_DYNAMIC_VALUES
 		Enables dynamically created simulator otuput as opposed
 		to static values.
 
+config GPS_SIM_MOVE_TIME
+	int "Time before new value generation"
+	default 10
```

Comment:
Good idea, will add that

Reply:
What unit is time expressed in? Is it seconds?

Reply:
I updated this with a more descriptive config name along with prompt and help text. 
Now it's called `GPS_SIM_FIX_TIME`, and the unit is milliseconds. It defaults to be the same as the trigger timer intervals because that timer creates `data_rdy` trigger events. If button is used to trigger that event, the default time for new position fix is set to 1 second.

Reply:
Thanks.